### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by Mujeeb4582**
## The following was implemented;

- The content of the email field has to be in lowercase.
- When the user submits the form, you check if the email is in lowercase.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the form is not sent.

### CODE REVIEW

File: form-validation.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  errorMessage += 'Form sent successfully';
+  errorInfo.innerHTML = `
+      <span style='color:green; border:1px solid green; padding: 8px;'>${errorMessage}</span>`;
+  return true;
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  validateEmail();
+});
```
Comment (Nemwel-Boniface) [inline form-validation.js]: - [ ] Good job working on the form validation it works perfectly fine :fire: . However, according to the project requirements, when the validation passes the form is supposed to submit to form spree. When I inspect the network tab when I click send, I cannot see the form being submitted. This is not the desired outcome as per the project requirements. Kindly make sure that when someone clicks send, it submits the form.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Mujeeb4582      :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: form-validation.js (hunk 1 above)
Comment (Mujeeb4582) [reply form-validation.js]: ## check the latest commits 
- I resolve the problems
- I add the function in form-validation.js file
- Now the form submitting to the desire server.

**Comment by Mujeeb4582** [comment]
## check the latest commits 
- I resolve the problems
- I add the function in form-validation.js file
- Now the form submitting to the desire server.

**Review by Deepakdanger — APPROVED** [review]
Hi @Mujeeb4582 , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**